docs/user/aws/install_upi.md

```diff
@@ -0,0 +1,432 @@
+# Install: User Provided Infrastructure (UPI)
+
+The steps for performing a UPI-based install are outlined here. Several [CloudFormation][cloudformation] templates are
+provided to assist in completing these steps or to help model your own.
+
+## Create Ignition Configs
+
+The machines will be started manually. Therefore, it is required to generate the bootstrap and machine Ignition configs
+and store them for later steps.
+
+```console
+$ openshift-install-linux-amd64 create ignition-configs
+? SSH Public Key /home/user_id/.ssh/id_rsa.pub
+? Platform aws
+? Region us-east-1
+? Base Domain example.com
+? Cluster Name openshift
+? Pull Secret [? for help]
+```
+
+After running the command, several files will be available in the directory.
+
+```console
+$ tree
+   .
+   ├── auth
+   │   └── kubeconfig
+   ├── bootstrap.ign
+   ├── master.ign
+   ├── metadata.json
+   └── worker.ign
+```
+
+## Create/Identify the VPC to be Used
+
+You may create a VPC with various desirable characteristics for your situation (VPN, route tables, etc.). The recommended
+VPC configuration and a CloudFormation template is provided [here](../../../upi/aws/cloudformation/01_vpc.yaml).
+
+A created VPC via the template or manually should approximate a setup similar to this:
+
+![Example VPC Architecture Diagram](images/install_upi_vpc.png)
```

Comment:
This graphic has "Master Subnet (public) ..." etc., which is no longer the case for IPI AWS since #1045.  And as of b29c9736a77 you have `05_cluster_master_nodes.yaml` content like:

```yaml
  Master0Subnet:
    Description: The subnets (recommend private) to launch the master nodes into
    Type: AWS::EC2::Subnet::Id
```

which suggests your CloudFormation configs are following the new IPI approach and putting all nodes in the private subnets.  Can you update your graphic to match?

Reply:
On the "update the graphic" approach, while I think this graphic is great for communicating the structure, I'm concerned about maintaining it.  Can you provide it as an SVG or similar to make it easier for others to edit it going forward?  Also, if there's a preferred tool (Inkscape?) for editing it, that would be good to document in the commit message.

Reply:
Updated graphic. It's a [Dia](http://dia-installer.de/) diagram. I could add the source material to a subdir. Path preference?

Reply:
> Path preference?

Right next to the PNG, as `docs/user/aws/images/install_upi_vpc.dia`?